lib/tasks/install-blueprint.js

```diff
@@ -44,24 +45,24 @@ class InstallBlueprintTask extends Task {
 
     installOptions = merge(installOptions, options || {});
 
-    return this._resolveBlueprint(blueprintOption).then(blueprint => {
+    return this._resolveBlueprint(installOptions).then(blueprint => {
```

Comment:
I think I would prefer it to not pass all of the options along. Either use a dedicated `yarn = true/false` parameter or an `options` parameter which only accepts/cares about `{ yarn: true/false }` for now. In other words, I think we should keep having `name` as a dedicated first parameter.